src/theia/sfm/camera/reprojection_error.h

```diff
@@ -46,20 +46,20 @@ struct ReprojectionError {
  public:
   explicit ReprojectionError(const Feature& feature) : feature_(feature) {}
 
-  template<typename T> bool operator()(const T* camera_parameters,
+  template<typename T> bool operator()(const T* camera_extrinsics, const T* camera_intrinsics,
```

Comment:
wrap the const T* camera intrinsics onto a new line, and align it with the other function parameters